e2e-cypress/Jenkinsfile.template

```diff
@@ -10,6 +10,7 @@ odsComponentPipeline(
   ]
 ) { context ->
   stageTest(context)
+  odsComponentStageScanWithSonar(context)
```

Comment:
Should we reverse the order? I think it is probably not needed to run the tests before doing the analysis, right? And tests will likely take quite a bit longer ...

Reply:
Agree with the reversed order. Running them after the SonarQube scan would also encourage you to do e2e tests locally to be more sure that that stage does not fail...

Reply:
does not work - for SQ you need binaries :(

Reply:
@clemensutschig This is TypeScript ... no binaries. IIRC SQ scanning works anytime against the source files only.